test/unit/config/test_load_config.py

```diff
@@ -29,71 +29,71 @@ def mock_process_config(self, kwargs):
+    assert config.raw_config['property6'] is None  # no change
 
-    assert type(config._raw_config['property1']) is str
-    assert type(config._raw_config['property2']) is int
-    assert type(config._raw_config['property3']) is float
-    assert type(config._raw_config['property4']) is bool
+    assert type(config.raw_config['property1']) is str
+    assert type(config.raw_config['property2']) is int
+    assert type(config.raw_config['property3']) is float
+    assert type(config.raw_config['property4']) is bool
 
 
-def test_update_raw_config_from_string_kwargs(mock_init):
+def test_updateraw_config_from_string_kwargs(mock_init):
     # kwargs may be passed as strings: property data types should not be affected
     config = GalaxyAppConfiguration(property1='b', property2='2', property3='2.0', property4='false')
 
-    assert len(config._raw_config) == 6  # no change
-    assert config._raw_config['property1'] == 'b'  # updated
-    assert config._raw_config['property2'] == 2  # updated
-    assert config._raw_config['property3'] == 2.0  # updated
-    assert config._raw_config['property4'] is False  # updated
+    assert len(config.raw_config) == 6  # no change
+    assert config.raw_config['property1'] == 'b'  # updated
+    assert config.raw_config['property2'] == 2  # updated
+    assert config.raw_config['property3'] == 2.0  # updated
+    assert config.raw_config['property4'] is False  # updated
 
-    assert type(config._raw_config['property1']) is str
-    assert type(config._raw_config['property2']) is int
-    assert type(config._raw_config['property3']) is float
-    assert type(config._raw_config['property4']) is bool
+    assert type(config.raw_config['property1']) is str
+    assert type(config.raw_config['property2']) is int
+    assert type(config.raw_config['property3']) is float
+    assert type(config.raw_config['property4']) is bool
 
 
-def test_update_raw_config_from_kwargs_with_none(mock_init):
+def test_updateraw_config_from_kwargs_with_none(mock_init):
     # should be able to set to null regardless of property's datatype
     config = GalaxyAppConfiguration(
         property1=None, property2=None, property3=None, property4=None, property5=None, property6=None,
     )
 
-    assert config._raw_config['property1'] is None
-    assert config._raw_config['property2'] is None
-    assert config._raw_config['property3'] is None
-    assert config._raw_config['property4'] is None
-    assert config._raw_config['property5'] is None
-    assert config._raw_config['property6'] is None
+    assert config.raw_config['property1'] is None
+    assert config.raw_config['property2'] is None
+    assert config.raw_config['property3'] is None
+    assert config.raw_config['property4'] is None
+    assert config.raw_config['property5'] is None
+    assert config.raw_config['property6'] is None
 
 
-def test_update_raw_config_from_kwargs_falsy_not_none(mock_init):
+def test_updateraw_config_from_kwargs_falsy_not_none(mock_init):
```

Comment:
Revert.

Reply:
The 4 comments above: done. Marking as resolved.